src/liquid/components/ld-toggle/test/ld-toggle.e2e.ts

```diff
@@ -254,6 +268,22 @@ describe('ld-toggle', () => {
           const results = await page.compareScreenshot()
           expect(results).toMatchScreenshot()
         })
+        it(`invalid disabled theme-${theme}${checkedStateStr}`, async () => {
+          const page = await getPageWithContent(
+            `<ld-toggle${checkedStateStr} required disabled />`,
+            theme
+          )
+          const results = await page.compareScreenshot()
+          expect(results).toMatchScreenshot()
+        })
+        it(`invalid aria-disabled theme-${theme}${checkedStateStr}`, async () => {
+          const page = await getPageWithContent(
+            `<ld-toggle${checkedStateStr} required aria-disabled="true" />`,
+            theme
+          )
+          const results = await page.compareScreenshot()
+          expect(results).toMatchScreenshot()
+        })
```

Comment:
In case of "checked" this is not invalid. This goes for the other added "invalid" state tests, as well.

Reply:
You are right. Though, I didn't change the naming of the test (specifically the _invalid_ part of it), if this is what you are referring to. I just added the _disabled_ variant to the tests. You could say that the test is specifically testing the component for how it looks for the invalid state being either true or false, thus testing the invalid state. Then the name makes sense. I would leave it as is.